Makefile

```diff
@@ -51,7 +51,7 @@ clean_databases:
 follow_logs:
 	-docker-compose -f docker-compose.yml logs -f
 
-test_integration: stop setup
+test_integration: setup
 	$(MAKE) create_networks
 	-${DOCKER_COMPOSE_INFRA} up -d
 	./.scripts/docker/wait-healthy.sh reviewer_postgres_1 20
```

Comment:
should use a container_name setting that can be defined in the docker-compose. eg:

```
    container_name: reviewer_postgres
```

This makes the container name predictable

Reply:
will raise a separate issue for that